pyrasterframes/python/MANIFEST.in

```diff
@@ -0,0 +1,4 @@
+include LICENSE.txt
```

Comment:
I get the following when running `python setup.py`

I think LICENSE.md is meant.

```
reading manifest file 'pyrasterframes.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'LICENSE.txt'
warning: no files found matching '*.jar' under directory 'pyrasterframes'
```

Reply:
Also wonder if there is a problem because the assembly jar is in directory pyrasterframes/target which is up a directory from where setup.py is?

Reply:
I'll fix this in a new branch I've already started with some additional reorg.

https://github.com/locationtech/rasterframes/pull/133